components/database/postgresql-10/Makefile

```diff
@@ -78,7 +79,7 @@ CONFIGURE_OPTIONS.64 += --libdir=/usr/postgres/$(COMPONENT_MJR_VERSION)/lib/$(MA
 
 CONFIGURE_OPTIONS+=$(CONFIGURE_OPTIONS.$(BITS))
 
-CONFIGURE_ENV	+= PYTHON=$(PYTHON)
+CONFIGURE_ENV	+= PYTHON=/usr/bin/python3.7
```

Comment:
You can keep the original
CONFIGURE_ENV += PYTHON=$(PYTHON) when you explicitly set the python version to use by setting
PYTHON_VERSION=		3.7
some lines above.